lib/rules/jsx-first-prop-new-line.js

```diff
@@ -35,7 +84,13 @@ module.exports = {
             if (decl.loc.start.line === node.loc.start.line) {
               context.report({
                 node: decl,
-                message: 'Property should be placed on a new line'
+                message: 'Property should be placed on a new line',
+                fix: function(fixer) {
+                  var nodeIndent = getNodeIndent(node, false, false);
```

Comment:
`getNodeIndent`'s 2nd and 3rd arguments are default to `false`, can remove here.